README.md

```diff
@@ -84,5 +84,17 @@ const handler = new LlrtFunction(this, 'Handler', {
 });
 ```
 
+### LLRT Layer
+
+By default, CDK Lambda LLRT bundles the LLRT bootstrap binary directly in your lambda function.
+If you want to package the binary in a separate layer that can be shared amongst functions then set `llrtLayer` to `true`.
+
+```ts
+const handler = new LlrtFunction(this, 'Handler', {
+    entry: 'lambda/index.ts',
+    llrtLayer: true
+});
+```
```

Comment:
One final comment: do we need a document about how to force update the binary? For example, when version is set as `latest`, it would be difficult to update the binary when docker caches an old version.

Previously we used some instruction about that

> If you want to upgrade the LLRT version, remove the `.tmp` directory, which contains the cache of LLRT binary fetched from GitHub (only applicable when you set `llrtVersion` to `latest` (default)).